doc/doxygen/code-gallery/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@
 
 ADD_CUSTOM_TARGET(code-gallery)
 
+MESSAGE(STATUS "Setting up code gallery documentation from ${DEAL_II_CODE_GALLERY_DIRECTORY}.")
```

Comment:
I always set it up, but the code gallery page is sometimes empty and only shows a terse message that there are no programs to show.